### PR TITLE
Small updates to date style from feedback #446

### PR DESCRIPTION
**Pull request by paulcpederson**
- hide next/prev buttons if out of range 🤸‍♂️ 
- fix baseline alignment of month + year in header 📏 
- make sure paddles are square 🚣
- optically center number in date circle  👁️
- remove shadow on always open calendar 👐

### CODE REVIEW

File: src/components/calcite-date/calcite-date.scss [hunk 1]
```diff
@@ -11,15 +11,15 @@
...
 }
 
 :host([scale="m"]) {
-  max-width: 288px;
+  max-width: 286px;
 }
 
 :host([scale="l"]) {
-  max-width: 400px;
+  max-width: 398px;
```
Comment (macandcheese) [inline src/components/calcite-date/calcite-date.scss:22]: 🤷‍♂️

Comment (paulcpederson) [reply src/components/calcite-date/calcite-date.scss:22]: pixel perfect, @macandcheese , pixel perfect

Screen Shot 2020-04-29 at 10 02 59 AM

Comment (macandcheese) [reply src/components/calcite-date/calcite-date.scss:22]: Ahh, 365 days in a year, 365 pixels tall. 
tenor

Comment (bstifle) [reply src/components/calcite-date/calcite-date.scss:22]: will need to change every leap year to 366px

**Pull request merged**